pkg/services/ngalert/store/alert_rule.go

```diff
@@ -275,9 +275,26 @@ func (st DBstore) ListAlertRules(ctx context.Context, query *ngmodels.ListAlertR
 		q = q.Asc("namespace_uid", "rule_group", "rule_group_idx", "id")
 
 		alertRules := make([]*ngmodels.AlertRule, 0)
-		if err := q.Find(&alertRules); err != nil {
+		var faultyRules int64 = 0
```

Comment:
```suggestion
		var faultyRules int64
```
Nitpick:This or `faultyRules := int64(0)` is the style that's more commonly used in the project